### PR TITLE
Make mop bucket destructible

### LINKED ISSUE #16122: Mop buckets are indestructible
## Description

The indomitable jani spirit.

### PR DESCRIPTION
**Pull request by notquitehadouken**
## About the PR

Fixes #16122 
Mop buckets now break and have half the health of a trolley

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- fix: Mop buckets are no longer indestructible

### CODE REVIEW

**Comment by Slava0135** [comment]
> hope you can make these somehow otherwise jani gonna have no buckets

bananium mop buckets

**Comment by Slava0135** [comment]
you can actually order buckets from cargo

**Comment by mirrorcult** [comment]
this should spill contents on destruction

**Comment by notquitehadouken** [comment]
> What about making cardboard boxes destructible too?

:stare:

**Comment by keronshb** [comment]
> What about making cardboard boxes destructible too?

They are, just not for syndies.

File: Resources/Prototypes/Entities/Objects/Specific/Janitorial/janitor.yml [hunk 1]
```diff
@@ -122,6 +122,27 @@
...
+          !type:DamageTrigger
+          damage: 200
+        behaviors:
+          - !type:DoActsBehavior
+            acts: [ "Destruction" ]
+      - trigger:
+          !type:DamageTrigger
+          damage: 100
+        behaviors:
+          - !type:EmptyAllContainersBehaviour
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Specific/Janitorial/janitor.yml:140]: This doesn't do anything as it doesn't have containers.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Comment by Myakot** [comment]
> 

image
No they are not destructible by bullets. First thind I tried doing with new ice anomaly was either blocking it's projectiles with mop buckets or boxes, boxes are larger and easier to use.
Sorry for off-topping.

**Comment by notquitehadouken** [comment]
alright i'll make a less horrible solution

**Pull request closed without merging**